functional/ek-cert-use-ek_check_script/test.sh

```diff
@@ -29,16 +31,22 @@ rlJournalStart
 
         rlRun "cat > /var/lib/keylime/check_ek_script.sh <<_EOF
 #!/bin/sh
-echo AGENT_UUID=$AGENT_UUID
-echo ENDORSEMENT_KEY=$EK
-echo ENDORSEMENT_KEY_TPM=$EK_TPM
-echo ENDORSEMENT_KEY_CERTIFICATES=$EK_CERT
-echo PROVKEYS=$PROVKEYS
+printenv | grep -w AGENT_UUID
```

Comment:
IMHO the former script version was find and in fact even faster. ;-)

Reply:
I'm not able to access environment variables in script, when I used former script. Variables are empty then. So I need to printenv and grep them.

Reply:
Thank you for good advice!